controllers/toolchainstatus/toolchainstatus_controller.go

```diff
@@ -490,20 +497,24 @@ func (r *Reconciler) sendToolchainStatusNotification(logger logr.Logger,
 	tsValue := time.Now().Format("20060102150405")
 	contentString := ""
 	subjectString := ""
+	domain := ""
+	if domain, err = removeSchemeFromURL(toolchainStatus.Status.HostRoutes.ProxyURL); err != nil {
+		logger.Error(err, fmt.Sprintf("Error while parsing proxyUrl %v", toolchainStatus.Status.HostRoutes.ProxyURL))
+		domain = toolchainStatus.Status.HostRoutes.ProxyURL
```

Comment:
I guess we want to not set the domain here when is invalid right?  as you did on line https://github.com/codeready-toolchain/host-operator/pull/870/files#diff-dbe76ee9dfabe95c78e8d2e7c3efdf6526bfd4719168b8529c518129cb83a6acR543
```suggestion
```